File: python homework/1.py
# 输入多个数字，求最大值、最小值、平均值
nums = list(map(float, input("请输入数字，用空格分开：").split()))

max_num = max(nums)
min_num = min(nums)
avg_num = sum(nums) / len(nums)

print("最大值：", max_num)
print("最小值：", min_num)
print("平均值：", avg_num)

File: cpp homework/1.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int n;
    cout << "请输入数字个数：";
    cin >> n;

    vector<double> nums(n);
    cout << "请输入 " << n << " 个数字：";
    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }

    // 最大、最小、平均
    double max_num = *max_element(nums.begin(), nums.end());
    double min_num = *min_element(nums.begin(), nums.end());
    double sum = 0;
    for (auto num : nums) sum += num;
    double avg = sum / n;

    cout << "最大值：" << max_num << endl;
    cout << "最小值：" << min_num << endl;
    cout << "平均值：" << avg << endl;

    return 0;
}